src/makeRobotAccountant.js

```diff
@@ -7,6 +7,21 @@
 
 function makeRobotAccountant() {
   // write code here
+  let callCounter = 0;
```

Comment:
[CODE STYLE] - You should use one variable for counting function calls, which you have correctly implemented here with `callCounter`.